### DISCUSSION TITLE
SOPS key rotation procedure

### DISCUSSION THREAD

**Discussion by Smana** [fluxcd/flux2#1079] [Q&A]
Hello,

I was just wondering if there's a procedure somewhere if you want to regularly rotate the key used to encrypt secrets ?
I wasn't able to find the answer. I don't know if we can have a similar system as Sealed secrets, which has a built-in rotation system.

Thank you

**Comment by vterdunov** [comment]
sops can rotate keys. https://github.com/mozilla/sops#id13
My procedure is to use one GPG key for Flux, and separate keys for admins and developers.
